packages/mdl-theme/README.md

```diff
@@ -0,0 +1,106 @@
+# mdl-theme
```

Comment:
I'd also maybe talk about _how_ theming is done, to give developers an idea of the flexibility and versatility of the package